src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double area() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Right Triangle, Color:" + color + ", First Leg: " + firstLeg + ", Second Leg: " + secondLeg + ", Area: " + area());
```

Comment:
Use 'System.out.format' or 'System.out.printf' for better readability instead of concatenation in 'System.out.println'.